src/main/java/com/neu/wham/services/LookupServiceImpl.java

```diff
@@ -0,0 +1,127 @@
+package com.neu.wham.services;
+
+import java.util.*;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class LookupServiceImpl implements LookupService {
+	
+	Map<String, String> abbvSource;
```

Comment:
Contains all the locations from - https://www.northeastern.edu/campusmap/map/qad4.html and select important ones from https://www.northeastern.edu/campusmap/map/index.html.